client/src/components/Post/Post.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../styles/variables';
```

Comment:
pls rename the file to style.scss

Reply:
I cannot rename 'variables' to 'style' that because we have another style.scss in styles folder